### PR TITLE
[DSS-285] Update Button story to show one story in sidebar

### PR DESCRIPTION
**Pull request by goodwinchris**
# Description

Update Sage-Button story to show only one default example of our button. For all variants we now add a `<sage-button />` to the page. This limits the number of button stories in the sidebar to 1. 

[Fixes DSS-285](https://kajabi.atlassian.net/browse/DSS-285)

## Type of change

Update to storybook

# How Has This Been Tested?

There should be no testing as the component hasn't been changed, only storybook is updated.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: libs/core/src/components/sage-button/stories/sage-button.stories.mdx [hunk 1]
```diff
@@ -43,80 +50,50 @@ When following our usage guidelines, sage-button should be accessible right out
...
-  <Story name="PrimaryDisabled" args={{ icon: '', disabled: true, slot: 'Disabled', type: 'button' }}>
-    {Template.bind()}
-  </Story>
-  <Story name="PrimaryIcon" args={{ icon: 'trashIcon', disabled: false, slot: 'Icon', type: 'submit' }}>
-    {Template.bind()}
-  </Story>
+<Canvas withSource="none">
+  <sage-button>Default</sage-button>
+  <sage-button disabled="true">Disabled</sage-button>
+  <sage-button icon="trashIcon">Icon</sage-button>
```
Comment (ju-Skinner) [inline libs/core/src/components/sage-button/stories/sage-button.stories.mdx]: Maybe I wasn't clear previously, but I was ok with multiple stories, e.g Primary, Secondary, Accent. I felt the slight variants `PrimaryIcon`, and `PrimaryDisabled` was a little too much.

Comment (goodwinchris) [reply libs/core/src/components/sage-button/stories/sage-button.stories.mdx]: Gotcha, updates pushed 👍

**Review by ju-Skinner — APPROVED** [review]

**Review by anechol — APPROVED** [review]
Now that I'm seeing this live, I really love the separation here. Looks super clean and organized, and we can still be explicit while giving the engineers the exact scenario they need. Nice!!

**Review by QuintonJason — APPROVED** [review]
Looks great!

**Review by ju-Skinner — APPROVED** [review]

**Pull request merged**